src/sentry/static/sentry/app/components/actions/ignore.tsx

```diff
@@ -276,48 +313,192 @@ export default class IgnoreActions extends React.Component<Props, State> {
+const disabledCss = css`
+  color: #ced3d6;
+  border-color: #e3e5e6;
+  box-shadow: none;
+  cursor: default;
+  opacity: 0.65;
+  pointer-events: none;
+`;
+
+const actionLinkCss = p => css`
+  color: ${p.theme.subText};
+  &:hover {
+    border-radius: ${p.theme.borderRadius};
+    background: ${p.theme.bodyBackground} !important;
+  }
+`;
+
+const dropdownTipCss = p => css`
+  & ul {
+    padding: 0;
+    border-radius: ${p.theme.borderRadius};
+    top: 40px;
+    &:after {
+      border-bottom: 8px solid ${p.theme.bodyBackground};
+    }
+  }
+`;
+
+const IgnoreWrapper = styled('div')`
+  display: inline-block;
+  ${dropdownTipCss};
+  & span {
+    position: relative;
+  }
+`;
+
+const IgnoredButtonActionWrapper = styled('div')`
+  margin-right: 5px;
+  display: inline-block;
+`;
+
+const StyledIgnoreButton = styled(Button)`
+  display: inline-flex;
+  vertical-align: middle;
+  color: ${p => p.theme.white};
+  background: ${p => p.theme.purple300};
+  border: 1px solid #4538a1;
+  & span {
+    padding: 6px 4.5px;
+  }
+  &:hover {
+    background: ${p => p.theme.purple300};
+    color: ${p => p.theme.white};
+    border: 1px solid #4538a1;
+  }
+`;
+
+const StyledIgnoreActionLink = styled(ActionLink)`
+  display: flex;
+  float: left;
+  color: #493e54;
+  background-image: linear-gradient(to bottom, ${p => p.theme.white} 0%, #fcfbfc 100%);
+  background-repeat: repeat-x;
+  box-shadow: 0 2px 0 rgba(0, 0, 0, 0.03);
+  font-size: ${p => p.theme.fontSizeSmall};
+  padding: ${space(0.5)} 9px;
+  border: 1px solid ${p => p.theme.border};
+  border-radius: 3px 0 0 3px !important;
+  border-right: 0;
+  font-weight: 600;
+  line-height: 1.5;
+  user-select: none;
+  transition: none;
+  ${p => (p.disabled ? disabledCss : null)};
+  &:hover {
+    background-color: #e6e6e6;
+    border-radius: 3px;
+    color: ${p => p.theme.button.default.color};
+    border-color: #afa3bb;
+    box-shadow: 0 2px 0 rgba(0, 0, 0, 0.06);
+  }
+`;
+
 const StyledIconNot = styled(IconNot)`
   margin-right: ${space(0.5)};
+  align-self: center;
   @media (max-width: ${p => p.theme.breakpoints[0]}) {
     display: none;
   }
 `;
 
-// The icon with no text label needs positioning tweaks
-// inside the bootstrap button. Hopefully this can be removed
-// bootstrap buttons are converted.
-const SoloIconNot = styled(IconNot)`
-  position: relative;
-  top: 1px;
+const StyledActionLink = styled(ActionLink)`
+  padding: 7px 10px !important;
+  ${actionLinkCss};
 `;
 
-const StyledActionLink = styled(ActionLink)`
-  display: flex;
-  align-items: center;
-  transition: none;
+const StyledForActionLink = styled(ActionLink)`
+  padding: ${space(0.5)} 0;
+  ${actionLinkCss};
 `;
 
 const StyledDropdownLink = styled(DropdownLink)`
   transition: none;
+  border-top-left-radius: 0 !important;
+  border-bottom-left-radius: 0 !important;
+`;
+
+const DropdownMenuItem = styled('li')`
+  :not(:first-child) {
+    border-top: 1px solid ${p => p.theme.border};
```

Comment:
Should this be innerBorder? The screenshots seem high contrast in the menus.

Reply:
This is what it looks like with `innerBorder` (the designs had this set for `border`) - @robinrendle thoughts?

<img width="731" alt="Screen Shot 2020-12-10 at 1 31 57 AM" src="https://user-images.githubusercontent.com/20312973/101753618-b82ec180-3a87-11eb-9349-75d7fab9e25d.png">
<img width="363" alt="Screen Shot 2020-12-10 at 1 32 12 AM" src="https://user-images.githubusercontent.com/20312973/101753639-bc5adf00-3a87-11eb-86f3-60842df1797c.png">

Reply:
I prefer the inner border as it is more subtle and looks like other striped dropdowns in the app (global selection header, saved issue search etc).

Reply:
cool, updated to innerBorder

Reply:
Yeah, this looks way better. ✨